### ISSUE TITLE
Test suite failure with pytest 5.1.0

### ISSUE THREAD

**Issue by peti** [jquast/blessed#109]
After update pytest from version 4.6.5 to 5.1.0, we see the following test suite failure in NixOS. Is that a known problem? We are using the latest release version of blessed.

~~~
=================================== FAILURES ===================================
___________________ test_nested_formattingstring_type_error ____________________

monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7fffee5d0a50>

    def test_nested_formattingstring_type_error(monkeypatch):
        """Test formatters.FormattingString raising TypeError."""
        from blessed.formatters import FormattingString
    
        # given,
        pstr = FormattingString(u'a-', u'n-')
        expected_msg = (
            "Positional argument #1 is {0} expected any of "
            .format(type(1)))
    
        # exercise,
        with pytest.raises(TypeError) as err:
            pstr('text', 1, '...')
    
        # verify,
>       assert expected_msg in '{0}'.format(err)
E       assert "Positional argument #1 is <class 'int'> expected any of " in '<ExceptionInfo TypeError tblen=2>'
E        +  where '<ExceptionInfo TypeError tblen=2>' = <built-in method format of str object at 0x7fffee6b3130>(<ExceptionInfo TypeError tblen=2>)
E        +    where <built-in method format of str object at 0x7fffee6b3130> = '{0}'.format

blessed/tests/test_formatters.py:156: AssertionError
======================== 1 failed, 283 passed in 11.01s ========================
~~~

**Comment by avylove** [comment]
This should be fixed by #108. Basically, now the exception value must be referenced explicitly as `err.value`

**Comment by peti** [comment]
OK. I had to cherry pick https://github.com/jquast/blessed/pull/108/commits/76a54d39b0f58bfc71af04ee143459eefb0e1e7b and https://github.com/jquast/blessed/pull/108/commits/aa94e01aed745715e667601fb674844b257cfcc9 to make the build succeed.

Thank you very much for the quick response.
